canteenWeb/forms.py

```diff
@@ -1,5 +1,6 @@
 from django import forms
 
+
 class UserForm(forms.ModelForm):
```

Comment:
What are these forms for?

Reply:
For Django Admin. That requires Forms.

Reply:
But he's still working on integrating our custom user model with Django Admin.